plugins/fuzzy.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: fuzzy
+spec:
+  platforms:
+  - uri: "https://github.com/d-kuro/kubectl-fuzzy/releases/download/v1.0.1/kubectl-fuzzy_1.0.1_darwin_amd64.tar.gz"
+    sha256: "22d882ea919f811c81e72a8bec53110fb9598f580a6e2bb1f82e2290ba06009c"
+    bin: kubectl-fuzzy
+    files:
+    - from: kubectl-fuzzy
+      to: .
+    - from: LICENSE
+      to: .
+    selector:
+      matchLabels:
+        os: darwin
+        arch: amd64
+  - uri: "https://github.com/d-kuro/kubectl-fuzzy/releases/download/v1.0.1/kubectl-fuzzy_1.0.1_linux_amd64.tar.gz"
+    sha256: "3b05ace0adfe071b5cb8c8b5af4760412c361170d6de9c3faa087b69a5aa2517"
+    bin: kubectl-fuzzy
+    files:
+    - from: kubectl-fuzzy
+      to: .
+    - from: LICENSE
+      to: .
+    selector:
+      matchLabels:
+        os: linux
+        arch: amd64
+  - uri: "https://github.com/d-kuro/kubectl-fuzzy/releases/download/v1.0.1/kubectl-fuzzy_1.0.1_windows_amd64.tar.gz"
+    sha256: "c141dfb18754d125b455c18e9dea95a1c14db45b34508e5c7ad1b3a85d84135a"
+    bin: kubectl-fuzzy.exe
+    files:
+    - from: kubectl-fuzzy.exe
+      to: .
+    - from: LICENSE
+      to: .
+    selector:
+      matchLabels:
+        os: windows
+        arch: amd64
+  version: "v1.0.1"
+  shortDescription: Fuzzy Finder Pod selector plugin. Currently "kubectl logs" and "kubectl exec" are supported.
+  description: |
+    Kubectl fuzzy is a kubectl plugin providing a Fuzzy Finder Pod selector.
```

Comment:
It's not clear to me what this plugin actually does. Is it really just for selecting pods by a fuzzy pattern? If that is indeed the case, I think this is not the right approach. The recommended way to select pods in Kubernetes is via labels. Selecting individual pods via fuzzy match seems pretty wonky to me.

My original assumption was that this plugin does some fuzzy magic on the output of some kubectl commands. Apparently, I misread this :/

FWIW, there are already a few plugins which focus on `exec` and `logs` commands. For example https://github.com/gabeduke/kubectl-iexec, https://gitlab.com/grzesuav/kubectl-mtail, https://github.com/boz/kail and probably more.

Reply:
Thanks for the comment. I will fix the tool description.

That's right, you can select an object in the label selector.
But in order to use the label selector, you need to know which label is being used on the target Kubernetes object.
I was too lazy to look up labels, so I decided to create this tool.
A fuzzy search allows you to select an object without having to find out which label is used for the Kubernetes object.

As for exec and logs, I know there are other tools for interactively selecting pods, but I think that fuzzy searches differentiate them from other tools.

I have also added support for kubectl describe yesterday.
I'm going to add more features to this tool in the future.
https://github.com/d-kuro/kubectl-fuzzy/releases/tag/v1.1.0